fe/free_energy_rabfe.py

```diff
@@ -32,20 +36,28 @@ def log(self):
             self.mol_name,
             "dG_complex_conversion (K complex)",
             self.dG_complex_conversion,
+            "dG_complex_conversion_err",
+            self.dG_complex_conversion_error,
             "dG_complex_decouple (E0 + A0 + A1 + E1)",
             self.dG_complex_decouple,
+            "dG_complex_decouple_err",
+            self.dG_complex_decouple_error,
             "dG_solvent_conversion (K solvent)",
             self.dG_solvent_conversion,
+            "dG_solvent_conversion_err",
+            self.dG_solvent_conversion_error,
             "dG_solvent_decouple (D)",
             self.dG_solvent_decouple,
+            "dG_solvent_decouple_err",
+            self.dG_solvent_decouple_error,
         )
 
     @classmethod
     def from_log(cls, log_line):
         """parse log line"""
         mol_name, rest = log_line.split("stage summary for mol: ")[-1].split(" dG_complex_conversion (K complex) ")
         tokens = rest.split()
-        value_strings = tokens[0], tokens[9], tokens[13], tokens[16]
+        value_strings = [tokens[i] for i in (0, 2, 11, 13, 17, 19, 22, 24)]
```

Comment:
* Seems easy to make a typo here
* Confirm: this will raise an error if attempting to read from an incompatible log version?

Reply:
I have confirmed that this raises an error if attempting to read from the older log version, would you like it as a test?

Reply:
Test probably isn't necessary -- thanks for confirming!